### ISSUE TITLE
[Testing 25.0] dicom_archive/dicomTar.pl and dicom_archive/updateMri_upload.pl

### ISSUE THREAD

**Issue by cmadjar** [aces/Loris-MRI#927]
- [x] run dicomTar.pl without the -database flag and ensure nothing gets inserted into the DB
- [x] run dicomTar.pl with the -database flag and check that entries got inserted into the DB in the tarchive* tables correctly
- [x] run dicomTar.pl with the -mri_upload_update flag on a DICOM directory that was not previously uploaded via the imaging uploader (ensure there is nothing in mri_upload table for that study before running the script) => ensure that an entry has been added to mri_upload for the uploaded DICOM study

**Comment by CamilleBeau** [comment]
I removed the record of one of the DICOMs from the database and then ran dicomTar.pl with the -mri_upload_update flag. The .tar file was created in the target directory, and data was inserted into `tarchive` and `mri_upload` tables, but I got the following message: 

`(loris-mri-python) lorisadmin@cbeaudoin-test-25:/opt/Loris-MRI/bin/mri/dicom-archive$ perl dicomTar.pl /data/Loris-MRI/data/tarchive/2018/ImagingUpload-14-30-gRhCrT /data/Loris-MRI/data/tarchive/2018 -profile prod -database -mri_upload_update
Constant subroutine NeuroDB::objectBroker::ConfigOB::CREATE_CANDIDATES redefined at /usr/lib/x86_64-linux-gnu/perl-base/constant.pm line 171.
`

I'm not sure if this is a fail or pass. Let me know if I should be testing it a different way!
